types/advocates.ts

```diff
@@ -25,4 +25,19 @@ interface Advocate {
   slackUsername?: string;
 }
 
-export { AdvocatesWorldRegion, ADVOCATES_WORLD_REGION_OPTIONS, Advocate };
+interface QampProject {
+  name: string;
+  url: string;
+  website?: string;
+  tags: string[];
```

Comment:
"tags" is a little bit too generic for me 🤔... In this case, I think that we could call it "cohort" since we already know we're only display cohorts as tags in the cards.

Also, we're only display one single cohort, so we don't need an array.

Reply:
we can update this, though look at this comment in Figma, from @HuangJunye 

> _The card should contain project title, project categories tags, mentors names, mentees names, collaborators (non-advocates team members) names, project gitihub link. Expand to see project descriptions?_

it mentions the possibility of having project category tags... maybe they should be separated?

Cohort in the top right corner
Project cagetories in its own row?

We're also exploring how we can update/improve the Ecosystem card, in [this conversation](https://github.com/Qiskit/qiskit.org/issues/2852#issuecomment-1545778983)

Reply:
I would suggest following the design decided for the ecosystem page, having:
- `cohort: string` on the top right corner. We should ask Gemma if a project belongs to only one cohort and, if it can be in more than one, we should have `cohort: string[]`
- `categories: string[]` on the row after the title for showing the categories

Reply:
We need to know if a project can belong to more than 1 cohort, if that is the case and we restrict the number of main tags to 1, we should probably rethink the card

Reply:
Per offline conversations, a project will only have one tag to show one cohort